tests/framework/extensions/provisioning/creates.go

```diff
@@ -369,6 +397,13 @@ func CreateProvisioningAirgapCustomCluster(client *rancher.Client, clustersConfi
 		}
 	}
 
+	if clustersConfig.PSACT == string(provisioninginput.RancherBaseline) {
+		err := clusters.CreateRancherBaselinePSACT(client, clustersConfig.PSACT)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Would this change any behavior in the airgap tests? Do we want to skip `CreateProvisioningRKE1AirgapCustomCluster` implementation? @igomez06

Reply:
no this should be fine. We should add this to rke1 as well.

Reply:
I think I initially left the airgap test out? Correct me if I'm wrong

Reply:
add it to rke1 as well.

Reply:
Added